### ISSUE TITLE
How do I use this library?

### ISSUE THREAD

**Issue by tfausak** [cowbell/active_model-errors_details#1]
I tried to use this library and immediately hit a problem.

``` rb
require 'active_model' # 4.2.0
require 'active_model/errors_details' # 1.1.0
# .../lib/active_model/errors_details.rb:62:in `<top (required)>': private method `include' called for ActiveModel::Errors:Class (NoMethodError)
```

Did I do something wrong?

**Comment by thiagocifani** [comment]
If you are trying to use this gem with ruby 2.0.0, it will not work properly. I was getting this error on heroku with ruby 2.0.0, when I specified on my gemfile a late version it worked fine. Try it, and let me know anything.

**Comment by tfausak** [comment]
That's probably it. I'm trying to use this with Ruby 2.0.0p481. Why doesn't this work with Ruby 2.0?

**Comment by thiagocifani** [comment]
I have no idea, just another user only, but I think they should be using some relative or require method specific to versions of ruby > 2.0.0.

**Comment by tfausak** [comment]
I don't think requiring more recent Rubies is appropriate since Rails 4.2 supports Ruby 1.9.3.

**Comment by morgoth** [comment]
@tfausak I just pushed a fix. Please try it and let me know if this fixes your issue.

**Comment by tfausak** [comment]
I'm sorry it took so long for me to get back to you. Your patch (aadc8997b698cdb7d970213008e07063583f6363) fixed my issue.

``` rb
require 'active_model' # 4.2.1
# => true
require 'active_model/errors_details' # aadc899
# => true

class Person
  include ActiveModel::Validations

  attr_accessor :name

  validates :name,
    presence: true
end

person = Person.new
person.valid?
# => false
person.errors.details[:name]
# => [{:error=>:blank}]
```

Thank you!

**Comment by tfausak** [comment]
Can you release the fix as a new version?

**Comment by morgoth** [comment]
@tfausak It was released just after the fix and is present in version 1.1.1

**Comment by tfausak** [comment]
:+1: thank you!
